app/views/items/show.html.erb

```diff
@@ -78,21 +78,21 @@
 
     <div class="details">
         <h6><strong>Description</strong></h6>
-        <%= @item.description %>
-      <ul>
-        <% unless @diets.nil? %>
-          <% @diets.each do |diet| %>
-            <li>It may contains: <%= diet.name %></li>
-          <% end %>
-        <% end %>
+        <%= @item.description.capitalize %>
 
-        <% unless @diets.nil? %>
-          <% @allergens.each do |allergen| %>
-            <li>Allergen: <%= allergen.name %></li>
-          <% end %>
+      <% unless @diets.nil? %>
+        <% @allergens.each do |allergen| %>
+          <p><i class="fas fa-ban"></i>&nbsp<%= allergen.name.capitalize %></li></p>
         <% end %>
```

Comment:
@MarlAndre Maybe we change the icon for the allergens as discussed?

Reply:
Icon is changed :) !

<img width="271" alt="tabs desktop" src="https://user-images.githubusercontent.com/15382972/173568545-6dd2a22c-f512-410f-b578-6cce7c874c71.PNG">